### PR TITLE
Port to Windows

### PR DESCRIPTION
**Pull request by zserg**
This PR is an attempt to run Elodie on Windows platform. The following problems was fixed:
1. File path strings was replaced with os.path.join in core and gui app
2. Windows has no possibility to change current time zone. So the tests was fixed to  make the time difference calculation (ugly way but nothing else came to my mind)
3. GUI app: windows notification area doesn't support drag-and-drop feature. I decided to add a drop event to gui start page to get the following workflow:

- click to icon in notification area brings the app start page (import window)

- the drag-and-drop files to import window switches the app to update mode

It's my first JS code so any criticism is welcome. Unfortunately I have no possibility to check this changes on OS X

### CODE REVIEW

File: elodie/geolocation.py [hunk 1]
```diff
@@ -161,9 +161,10 @@ def reverse_lookup(lat, lon):
 
     try:
         params = {'format': 'json', 'key': key, 'lat': lat, 'lon': lon}
+        headers = {"Accept-Language": "en-US,en;q=0.5"}
```
Comment (jmathai) [inline elodie/geolocation.py]: Does this impact the response? If so we should put it in `constants.py` and later make it configurable.

File: elodie/tests/filesystem_test.py [hunk 2]
```diff
@@ -12,13 +15,26 @@
 from elodie.media.media import Media
 from elodie.media.photo import Photo
 from elodie.media.video import Video
+from nose.plugins.skip import SkipTest
 
 os.environ['TZ'] = 'GMT'
 
+if os.name == 'nt':
```
Comment (jmathai) [inline elodie/tests/filesystem_test.py]: Mind adding a comment for this block of code?

File: elodie/tests/filesystem_test.py [hunk 3]
```diff
@@ -12,13 +15,26 @@
...
+    tz_shift = (datetime.fromtimestamp(0) -
+                datetime.utcfromtimestamp(0)).seconds/3600
+else:
+    tz_shift = 0
+
+def path_tz_fix(s_path):
```
Comment (jmathai) [inline elodie/tests/filesystem_test.py]: Seems like this should go into the helper module. Thoughts?

File: elodie/tests/filesystem_test.py [hunk 4]
```diff
@@ -47,6 +63,8 @@ def test_create_directory_recursive_success():
     shutil.rmtree(folder)
 
 def test_create_directory_invalid_permissions():
+    if os.name == 'nt':
```
Comment (jmathai) [inline elodie/tests/filesystem_test.py]: In `helper.py`?
```
def is_windows():
    return os.name == 'nt'
```

**Comment by jmathai** [comment]
Looks great. Added a few comments. Thanks for the PR.

File: elodie/geolocation.py (hunk 1 above)
Comment (zserg) [inline elodie/geolocation.py]: Accepted-Language impacts the response. My system language is Russian and without this fix I get the place names in Russian, but Elodie doesn't work with non-ascii symbols. 
As you proposed,    Accepted-Language is moved to `constants.py`

File: elodie/tests/filesystem_test.py (hunk 4 above)
Comment (zserg) [inline elodie/tests/filesystem_test.py]: Done

**Comment by jmathai** [comment]
@zserg Finally merged this. Thanks!

**Comment by SamHasler** [comment]
@zserg how did you install `exiftool pyexiv2` on windows?

**Comment by zserg** [comment]
@SamHasler, I have got exiftool from http://www.sno.phy.queensu.ca/~phil/exiftool/install.html and 
windows installer for pyexiv2 from  http://tilloy.net/dev/pyexiv2/download.html

**Comment by jmathai** [comment]
@zserg We should include that in the Readme file.

**Pull request merged**